lms/static/js/student_account/views/account_settings_view.js

```diff
@@ -67,7 +68,9 @@
 
                 _.each(view.$('.account-settings-section-body'), function (sectionEl, index) {
                     _.each(view.options.tabSections[view.activeTab][index].fields, function (field) {
-                        $(sectionEl).append(field.view.render().el);
+                        if (field.view.enabled) {
```

Comment:
Curious: Is the plan to keep the "enabled" field around even after the feature flag is removed?  Or would this code also be removed with the feature flag?